Mlem/Views/Shared/Moderation/ModToolSheet.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ModToolSheet.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2024-02-14.
+//
+
+import Foundation
+import SwiftUI
+
+struct ModToolSheet: View {
+    let tool: ModTool
+    
+    var body: some View {
+        switch tool {
+        case let .instanceBan(user, shouldBan):
+            BanUserView(user: user, community: nil, shouldBan: shouldBan, postTracker: nil) // TODO: add
```

Comment:
It is used, but it's currently only accessible via the user profile page navigation menu--I plan to add admin actions to the context menu as well, at which point it will be much more accessible.